package.json

```diff
@@ -249,7 +249,7 @@
   },
   "private": true,
   "dependencies": {
-    "@department-of-veterans-affairs/component-library": "^3.3.4",
+    "@department-of-veterans-affairs/component-library": "^3.5.0",
```

Comment:
Should we just bump this to ^3.6.0? That’s what it looks like is being installed anyways

Reply:
yep just did that